### DISCUSSION TITLE
how to yank from documentation hover pop up?

### DISCUSSION THREAD

**Discussion by josephs07** [71/dance#312] [Q&A]
How to yank contents from documentation hover?
VS code by default lets us copy contents from the documentation popup. While using dance this behavior is not working. This is necessary while using languages that support keyword arguments. 

![image](https://github.com/71/dance/assets/23273684/126da9c1-d825-4175-a7fc-d54687ec159e)

**Comment by 71** [comment]
Discussion continues in issue #313 since it can be considered a feature request / bug.
